subsets/life-stages-composite.obo

```diff
@@ -1451,12 +1467,12 @@ is_a: BFO:0000004 ! independent continuant
 is_a: CHEBI:138675 ! gas molecular entity
 is_a: CHEBI:35107 ! azane
 is_a: CHEBI:37176 ! mononuclear parent hydride
-relationship: has_role CHEBI:228364 ! NMR chemical shift reference compound
-relationship: has_role CHEBI:50910 ! neurotoxin
-relationship: has_role CHEBI:59740 ! nucleophilic reagent
-relationship: has_role CHEBI:75771 ! mouse metabolite
-relationship: has_role CHEBI:77941 ! EC 3.5.1.4 (amidase) inhibitor
-relationship: has_role CHEBI:78433 ! refrigerant
+relationship: RO:0000087 CHEBI:228364 ! has role NMR chemical shift reference compound
```

Comment:
For information: I believe those changes, and all similar changes in that file, are a normal and expected consequence of 61e7f2f. We no longer automatically create “shorthands” (such as `has_role`) for every single RO relation, so in the OBO release products the relations appear with their CURIEs instead.

Reply:
Likewise for all the similar changes in `uberon.obo`.